### DISCUSSION TITLE
TextWatermarkOptions Location option addition

### DISCUSSION THREAD

**Discussion by igsuffolk** [LazZiya/ImageResize#22] [Ideas]
Great package, very handy and easy to use.

Have a requirement to add text at a specific position on an image.

Would be great to be able to specify x,y position in addition to the TargetSpot enums.

Other idea is to have Margin Left and Margin Top option in addition or instead of Margin to accomplish same thing.

**Comment by LazZiya** [comment]
Thanks for the kind comments and ideas. I will implement them once I have time :)
